09_timer/timer.rb

```diff
@@ -0,0 +1,25 @@
+class Timer
+  attr_writer :time
+  attr_writer :seconds
+
+  def seconds(seconds = 0)
+    @seconds = seconds
+  end
+
+  def padded(x)
+    x > 9 ? x.to_s : "0" + x.to_s
+  end
+
+  def time_string
+    h = @seconds.div(3600)
```

Comment:
Why use `div` vs the clasical `/` or `%` the mathematical notification is usually easier to read?

Also try to avoid [magic numbers](https://sourabhchablog.wordpress.com/2013/10/07/code-smell-of-the-week-magic-numbers/) they are considered a code smell.